tests/manage/monitoring/conftest.py

```diff
@@ -712,3 +712,51 @@ def do_nothing():
     test_file = os.path.join(measurement_dir, 'measure_workload_idle.json')
     measured_op = measure_operation(do_nothing, test_file)
     return measured_op
+
+
+@pytest.fixture
+def measure_stop_rgw(measurement_dir, request, rgw_deployments):
+    """
+    Downscales RGW deployments, measures the time when it was
+    downscaled and monitors alerts that were triggered during this event.
+
+    Returns:
+        dict: Contains information about `start` and `stop` time for stopping
+            RGW pods
+
+    """
+    oc = ocp.OCP(
+        kind=constants.DEPLOYMENT,
+        namespace=config.ENV_DATA['cluster_namespace']
+    )
+
+    def stop_rgw():
```

Comment:
won't this function be usable by other things? maybe we can take it out to be somewhere under `ocs_ci/`?

Reply:
This function just downscales rgw deployments and waits 5 minutes. This seems like a very specific scenario for monitoring. I would keep it here until similar functionality is needed in other tests.